openedx/features/course_experience/__init__.py

```diff
@@ -72,7 +72,17 @@
     __name__,
 )
 
-# Waffle flag to enable relative dates for course content
+# .. toggle_name: course_experience.relative_dates
+# .. toggle_implementation: ExperimentWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable relative dates for course content. A 'Dates' tab will visible in the
```

Comment:
@BbrSofiane I think this is missing a word right? "tab will be visible"?

Reply:
Indeed, fixed!